doc/source/development/contributing_code_style.rst

```diff
@@ -0,0 +1,199 @@
+.. _Not_sure_what_to_put_here:
```

Comment:
Really not sure.

Reply:
You can just name this `contributing_code_guide` and use that as the target in `contributing.rst` To address your question above as well